crates/storage/src/store/multistore.rs

```diff
@@ -0,0 +1,69 @@
+use std::sync::Arc;
+
+use super::substore::SubstoreConfig;
+
+/// A collection of substore, each with a unique prefix.
+#[derive(Debug, Clone)]
+pub struct Multistore {
```

Comment:
Should this be called `MultistoreConfig`, since it just has the configs and isn't a store in itself?